src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -426,11 +431,200 @@ const formConfig = {
+                      'ui:title': 'Document name'
+                    }
+                  }),
+                  { 'ui:description': documentDescription }
+                )
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    privateRecords: {
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        required: ['name', 'attachmentId'],
+                        properties: {
+                          name: {
+                            type: 'string'
+                          },
+                          size: {
+                            type: 'integer'
+                          },
+                          confirmationCode: {
+                            type: 'string'
+                          },
+                          attachmentId: {
+                            type: 'string',
+                            'enum': [
+                              '1',
+                              '2',
+                              '3',
+                              // '4',
+                              '5',
+                              '6'
+                            ],
+                            enumNames: [
+                              'Discharge',
+                              'Marriage related',
+                              'Dependent related',
+                              // 'VA preneed form',
+                              'Letter',
+                              'Other'
+                            ]
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        documentUpload: {
+          title: 'Lay statements or other evidence',
+          depends: (formData, index) => {
+            const hasOtherEvidence = _.get(`disabilities.${index}.view:otherEvidence`, formData);
+            return hasOtherEvidence;
+          },
+          path: 'supporting-evidence/:index/additionalDocuments',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          uiSchema: {
+            disabilities: {
+              items: {
+                additionalDocuments: Object.assign({},
+                  fileUploadUI('Lay statements or other evidence', {
+                    allowRename: true,
+                    itemDescription: 'Adding additional evidence:',
+                    endpoint: '/v0/preneeds/preneed_attachments', // TODO: update this with correct endpoint (e.g. '/v0/21-526EZ/medical_records')
+                    addAnotherLabel: 'Add Another Document',
+                    fileTypes: ['pdf', 'jpg', 'jpeg', 'png'],
```

Comment:
Are these TBD or are we sure these are the ones?

Reply:
Good question, these are in the mock, but I'll include a comment to follow up on this.